src/components/Identity/createIdSyncs.js

```diff
@@ -90,3 +90,20 @@ export default ({ destinations, config, logger, cookieJar }) => {
     });
   }
 };
+
+const createExpiryChecker = cookieJar => () => {
+  const nowInHours = Math.round(
```

Comment:
I'd like to see some unit tests on this logic.  From looking at it, it will take 90 minutes for a 1 hour expiration to expire.  You could replace `Math.round` with `Math.ceil`

Reply:
Good catch, I will create a testing ticket.